src/Symfony/Component/Cache/Adapter/PhpArrayAdapter.php

```diff
@@ -262,42 +262,4 @@ private function generateItems(array $keys)
             }
         }
     }
-
-    /**
-     * @throws \ReflectionException When $class is not found and is required
-     *
-     * @internal
-     */
-    public static function throwOnRequiredClass($class)
```

Comment:
we should keep this for now, to keep cross-component compat.
let's mark the method `@internal to be removed in Symfony 5.0` instead